src/features/amUI/common/PageLayoutWrapper/useGlobalMenu.tsx

```diff
@@ -34,10 +35,34 @@ const getAccountType = (type: string): 'company' | 'person' => {
   return type === 'Organization' ? 'company' : 'person';
 };
 
+const linkUrls = {
+  forms: {
+    no_nb: 'skjemaoversikt',
+    no_nn: 'nn/skjemaoversikt/',
+    en: 'en/forms-overview/',
+  },
+  about: {
+    no_nb: 'nyheter/om-nye-altinn',
+    no_nn: 'nn/nyheiter/om-nye-altinn',
+    en: 'en/news/About-the-new-Altinn',
+  },
+  'start-business': {
+    no_nb: 'starte-og-drive',
+    no_nn: 'nn/starte-og-drive',
+    en: 'en/start-and-run-business',
+  },
+  help: {
+    no_nb: 'hjelp',
+    no_nn: 'nn/hjelp',
+    en: 'en/help',
+  },
+};
+
```

Comment:
You shouldn't have to handle the language-difference in the urls here.
This is already handled in the `getAltinnStartPageUrl` function, which should be used instead of the `getAltinnInfoUrl` function.

(Though we could probably give them better names to make this more clear) 😅